java/amazon-kinesis-producer/src/main/java/com/amazonaws/services/kinesis/producer/KinesisProducerConfiguration.java

```diff
@@ -1439,6 +1479,46 @@ public KinesisProducerConfiguration setVerifyCertificate(boolean val) {
         return this;
     }
 
+    /**
+     * If you have users going through a proxy, set the host here.
+     *
+     * <p><b>Default</b>: ""
+     */
+    public KinesisProducerConfiguration setProxyHost(String val) {
+        proxyHost = val;
+        return this;
+    }
+
+    /**
+     * If you have users going through a proxy, set the port here.
+     *
+     * <p><b>Default</b>: 8080
```

Comment:
If we are not keeping a default proxyHost do we need to keep this port default as well ? We should rather allow the customers to override it when they need to use it rather than rely on the default

Reply:
kept it consistent with kinesis port and cloudwatch port and defaulted to 443